### PR TITLE
Run build.sh - autoconf and automake

### PR DESCRIPTION
**Pull request by gbprz**
Update autoconf and automake generated files after recent Ubuntu Bionic changes.

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.14.1 from Makefile.am.
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```
Comment (igorpeshansky) [inline Makefile.in]: Should we remove the `autom4te.cache` directory and put it in `.gitignore` instead?

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: Makefile.in (hunk 1 above)
Comment (gbprz) [reply Makefile.in]: Sounds safe to me after reading through: https://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/Autom4te-Cache.html

**Review by qingling128 — APPROVED** [review]
LGTM

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**